### ISSUE TITLE
Ill-defined evaluation on XLM

### ISSUE THREAD

**Issue by McSinyx** [McSinyx/viwikipi#1]
As raised from `sklearn.metrics.classification`:

    UndefinedMetricWarning: F-score is ill-defined and being set to 0.0 due to no predicted samples.

I have not investigate this any further.

**Comment by McSinyx** [comment]
This is due to too fast learning rate.  After adjusting Adam epsilon (to 0.0004) and rate (to 0.000001), xlm-mlm-17-1280 converges just fine.  Closing this issue for now.
